src/components/navigation/Breadcrumbs.tsx

```diff
@@ -81,15 +90,30 @@ const Breadcrumb = styled.a`
   @media (max-width: ${props => props.theme.breakpoints.sm}) {
     display: none;
   }
+
+  @media (min-width: ${props => props.theme.breakpoints.md}) {
+    &:after {
+      content: ">";
+      color: ${props => props.theme.colors.lightgray};
+      position: absolute;
+      margin-left: 13px
+    }
+  }
 `
 
 const BreadcrumbLast = styled(Breadcrumb)`
+
+  &:after {
+    display: none !important;
```

Comment:
*theoretically* the extended styles should override existing ones. If this `!important` is necessary, consider adding a comment explaining why

Reply:
overwrite works, just forgot to remove. thanks!